packages/@tests/marine/node/package.json.skip

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
could you explain one more time why package.skip.json?

Reply:
fixed that